.github/README.md

```diff
@@ -0,0 +1,126 @@
+| [@JarredStanford](https://github.com/JarredStanford)      | [@DerekSchwantner](https://github.com/DerekSchwantner)      | [@DaftBeowulf](https://github.com/DaftBeowulf)       | [@AngXuDev](https://github.com/AngXuDev)   | [@danpatrickoneill](https://github.com/danpatrickoneill) |
+
+## Project Overview
+
+[Trello Board](https://trello.com/b/2L0hcHSc/labs-15-allegiance)
+
+[Product Canvas](https://www.notion.so/Allegiance-2973bfcc17f34daa8f6786dd2ad2146d)
+
+[UX Design files](https://www.figma.com/file/2IOo1GVvRaR7FkxpeU8xFq/%F0%9F%8F%80-Allegiance?node-id=0%3A12)
+
+Allegiance is a sports social media platform that allows sports fans to to support their favorite teams, talk about sports, and interact with other fans.
+
+### Key Features
+
+- Users can view, join and create groups based on allegiances and location
+- Users can choose and display their allegiances on profile page
+- Groups feature posting, replying, likes, etc. for lively sports discussion
+- Group privacy controls and moderation controls
+- User-specific activity feed and notifications
+
+## Tech Stack
+
+### Front end built using:
+
+#### React
+
+🚫 Why did you choose this framework?
+
+- React's virtual DOM in ReactJS makes for faster development and better user experience
+- Designed for an app like ours featuring various configs of reusable user-specific components
+
+#### Front end deployed to `Netlify`
+
+#### [Back end](https://github.com/labs15-allegiance/backend/) built using:
+
+#### 🚫 Node/Express
+
+- Allows for quick building of scalable network applications
+- REST APP is predictable, and doesn't expose database architecture to FE
+- Very clean, DRY code using packages/middleware for validation, error handling, etc.
+
+# APIs
+
+## Auth0
+
+Allegiance uses [Auth0](https://auth0.com/) for its authentication and API call processes. Please find Auth0 documentation [here](https://auth0.com/docs), along with the step by step tutorial used in this project [here](https://auth0.com/docs/quickstart/spa/react).
+
+## Misc API here
+
+🚫Replace text below with a description of the API
+
+You can do anything your heart can imagine. In life you need colors. This is where you take out all your hostilities and frustrations. It's better than kicking the puppy dog around and all that so. I'm sort of a softy, I couldn't shoot Bambi except with a camera. Trees get lonely too, so we'll give him a little friend. We'll lay all these little funky little things in there.
+
+# Environment Variables
+
+In order for the app to function correctly, the user must set up their own environment variables. There should be a .env file containing the following:
+
+    *  REACT_APP_DOMAIN - Can be found in Auth0 account info
+    *  REACT_APP_CLIENTID - Can be found in Auth0 account info
+    *  REACT_APP_AUDIENCE - Can be found in Auth0 account info
+    *  REACT_APP_AUTHURL - URL variable for auth calls
+    *  REACT_APP_BASEURL - Staging server URL
```

Comment:
BASEURL is now the production api URL (didn't make sense to point the last version of the repo to a staging url) and PRODURL has been renamed to STAGEURL (although it's fine to take it out all together)